test/integration_test_helper.rb

```diff
@@ -124,6 +124,13 @@ def set_request_user(user)
     user = users(user) unless user.is_a?(User)
     create_cookie('test_user', user.login)
   end
+
+  def with_controller_caching(*controller_klasses)
+    controller_klasses.each { |c| c.perform_caching = true }
+    yield
+  ensure
+    controller_klasses.each { |c| c.perform_caching = false }
+  end
 end
```

Comment:
I don't think file or memory cache will make any difference. The Rails cache is used in a number of other, unrelated places in Foreman and libraries, not only this test (this test is exercising fragment caching only).